installing/installing_azure/installing-azure-account.adoc

```diff
@@ -20,6 +20,11 @@ in the Azure documentation.
 
 include::modules/installation-azure-limits.adoc[leveloffset=+1]
 
+[role="_additional-resources"]
+.Additional resources
+
+* For more information about optimizing storage for performance, see xref:../../scalability_and_performance/optimizing-storage.adoc#optimizing-storage[Optimizing storage].
```

Comment:
The extra text around the xref should be [minimized](https://github.com/openshift/openshift-docs/blob/main/contributing_to_docs/doc_guidelines.adoc#additional-resources-sections).

```suggestion
* xref:../../scalability_and_performance/optimizing-storage.adoc#optimizing-storage[Optimizing storage].
```